src/Microsoft.ML.Tokenizers/Model/SentencePieceTokenizer.cs

```diff
@@ -86,6 +86,8 @@ private SentencePieceTokenizer(ModelProto modelProto, IReadOnlyDictionary<string
                     _specialTokensReverse.Add(item.Value, item.Key);
                 }
 
+                // We create this Regex object without a timeout because we know it will perform the match operation in O(N) time complexity.
+                // Therefore, confidently process any text within a reasonable time.
```

Comment:
I'd remove this last sentence. The concern isn't the total time required but whether a small amount of input data could result in a disproportionate amount of conputation required.

Reply:
I'll remove the sentence.